src/core/utils/i18n.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+import type { I18nType } from 'core/types/i18n';
+
+type GetLocalizedTextWithLinkPartsParams = {|
+  i18n: I18nType,
+  text: string,
+  linkStart?: string,
+  linkEnd?: string,
+|};
+
+export const getLocalizedTextWithLinkParts = ({
+  i18n,
+  text,
+  linkStart = 'linkStart',
+  linkEnd = 'linkEnd',
+}: GetLocalizedTextWithLinkPartsParams): Object => {
+  const linkEnds = '__LINK__';
```

Comment:
I find that `linkEnds` sounds too much like `linkEnd`, which makes it a bit confusing. This is really just used as a delimiter for `split`, right? So maybe use the word `delimiter` in the variable name?